components/pdfviewer/annotations.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Annotations
+page_title: PdfViewer - Annotations
+description: Annotations in the PDF Viewer for Blazor. How to add and configure annotations in the PDF Viewer for Blazor
+slug: pdfviewer-annotations
+tags: telerik,blazor,pdf,pdfviewer,annotations
+published: True
+position: 15
+---
+
+# PdfViewer Annotations
+
+The PdfViewer supports a set of annotation types to enhance PDF documents. For example, users can highlight important text areas or add notes as free text.
+
+The PDFViewer creates a dedicated layer for the annotations and includes them when downloading or printing the file. If the user opens a file with annotations, they can edit only the annotations that were added using the PDFViewer `Annotations` tool. The PDFViewer does not support editing annotations added in a different PDF editor.
+
+To add and manage the annotations, the user can click on the dedicated button in the [PDF Viewer Toolbar]({%slug pdfviewer-toolbar%}). It opens an annotations bar with the available annotation types.
+
+## Supported Annotations
+
+The PDFViewer provides the following annotation types:
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Annotation Type | Description |
+| --- | --- |
+| **Text Highlight** | Allows highlighting the selected text in the document. Users can choose the desired highlight color. |
+| **Free Text** | Allows creating an element with free text anywhere in the document. Provides option to select the desired text color and font size. |
+
+## Working with Annotations
+
+**To manage the annotations the user should:**
```

Comment:
I am not entirely opposed to the bold formatting here, but do we need it?

Reply:
I want to visually distinguish these parts but we've agreed not to use cations for that purpose.